perfscale_regression_ci/scripts/scalability/replace_nodes/clouds/alicloud.sh

```diff
@@ -0,0 +1,20 @@
+export CLUSTER_NAME=$(oc get machineset -n openshift-machine-api -o=go-template='{{(index (index .items 0).metadata.labels "machine.openshift.io/cluster-api-cluster" )}}')
```

Comment:
Let's create a more common env.sh file for items like this that are consistent among all cloud types, this way if we ever need to edit we can in one place